src/js/edu-benefits/components/NavButtons.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+// TODO update this when progress button moves
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+
+import { getSectionList } from '../utils/sections';
+
+export default class NavButtons extends React.Component {
```

Comment:
Yeah, it's pretty much the same code. I'd have to tweak it slightly to make it reusable, but we could definitely do that.